canvas_modules/common-canvas/src/common-properties/components/field-picker/field-picker.jsx

```diff
@@ -125,6 +126,23 @@ export default class FieldPicker extends React.Component {
 		return filters;
 	}
 
+	/**
+	* Returns array of visible values in second column
+	*/
+	getCheckboxLabels() {
+		const fields = this.getVisibleData();
+		const labels = [];
+		for (let i = 0; i < fields.length; i++) {
+			const field = fields[i];
+			if ("origName" in field) {
```

Comment:
Should it display the `name` instead? For cases where there are duplicate field names in multiple schemas, users wont know which schema the field is in using `origName`